README.md

```diff
@@ -1,6 +1,12 @@
-# SpeedRunMod
-hello world
+# TerrariaSpeedrunVerificationMod
 
-Hi BlobDylan!
+Features:
+
+-by having the mod installed the user has a built in timer, when activated, a hash key will be set and displayed verifying the run. if the timer is stopped the key goes away, the key will also be somewhat transparent and change every once in a while making it harder to tamper with.
```

Comment:
Should the key change every once in a while? Do you think that'd make it harder or easier to verify a run?

Reply:
It will make it both harder and more accurate. the thing is that if somone is willing to do the editing to that extent it probably won't matter anyway so maybe make it change when bosses are killed or anything that happens once in a long time, or alternitavely have it remain the same and never change. I'm kind of indifferent about this one

Reply:
harder to verify. because the mod has to check all the keys right?

Reply:
Yea, I was thinking it'd be harder to verify and easier to splice, since you can just say "well at 10:00 that's what it changed to. Kind of defeats the purpose of having 1 key per run. I'd advise against it

Reply:
alright agreed